pkg/types/ovirt/config.go

```diff
@@ -0,0 +1,75 @@
+package ovirt
```

Comment:
This (or most of this) should probably go in pkg/assets/...

Reply:
+1